### DISCUSSION TITLE
Code that shouldn't go public, should neither be kept as a commit.

### DISCUSSION THREAD

**Discussion by janoglezcampos** [hxr404/Discord-Console-hacks#54] [General]
The nitro hack code is still in the repo, even if you delete it in the last commits. Idk if you deleted it because it didn't work or because you didn't want it to go public since the code replacement changes its content through commits, but if it really works and you really want it not to go public, you should be aware of this.

**Comment by hxr404** [comment]
Thanks for Informing me, but I removed it because it doesn't work and because I realized that labeling the emoji module as "Nitro Hack" is kinda misleading. If I didn't want it to go public, I wouldn't have commited it in the first place.

**Comment by RobinRMC** [comment]
This thread could be locked, as there is no reason to let people respond to this. The conversation is done and doesn't need any further replies.

**Reply by hxr404** [reply]
Dude no offense, but like if you wouldn't have commented on this, there probably wouldn't have been any more comments (yeah, the conversation really doesn't need any further replies).

Also usually you don't lock conversations on GitHub just because they are "done", it is used as a moderation tool / or to move issues/discussions somewhere else.

Marking this as off-topic and locking the conversation now for good. OP probably doesn't want to get pings from this meta discussion.

Feel free to ping me on matrix if you really want to continue talking about this, though.
